app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :bet
+
+  validates_presence_of :body
+  validates_presence_of :user
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.